user-guide/content/docs/installation.md

```diff
@@ -0,0 +1,97 @@
+---
+weight: 3
+title: Installation Instructions
+---
+
+## Using Docker (recommended)
```

Comment:
Hm, if that's so, more effort needs to be put in testing this.

Reply:
What type of efforts do you imagine? Would you be able to capture these as stories and discuss it with Alex?